src/components/shared/NavBar.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+const NavBar = () => (
+  <header>
+    <nav className="flex space-btn">
+      <ul className="navbar flex space-btn">
+        <li className="nav-link">
+          <Link to="/">BookStore CMS</Link>
+        </li>
+        <li className="nav-link">
+          <Link to="/">BOOKS</Link>
+        </li>
```

Comment:
- [ OPTIONAL] It is not necessary to have two links in the navbar which lead to the same page "Book page" 🤔  I think you only need to have <Link>s in the navbar which leads to the booklist and categories locations 👍🏻